webapp/src/shared/components/Header/Header.tsx

```diff
@@ -0,0 +1,50 @@
+import { Link } from 'react-router-dom';
+import { Color } from '../../types';
+import Icon, { IconSize, IconVariant } from '../Icon/Icon';
+import Status, { StatusVariant } from '../Status/Status';
+import './Header.css';
+
+type HeaderProps = {
+  navigation: IconVariant | string;
+  urlNavigation: string;
+  status?: StatusVariant;
+  children: string;
+};
+
+export default function Header({
+  navigation,
+  urlNavigation,
+  status,
+  children,
+}: HeaderProps) {
+  console.log(typeof navigation);
+  return (
+    <header className="header">
+      <div className="header-navigation">
+        {navigation in IconVariant ? (
+          <Link to={urlNavigation}>
+            <Icon
+              variant={navigation as IconVariant}
+              size={IconSize.SMALL}
+              color={Color.LIGHT}
+            />
+          </Link>
+        ) : (
+          <Link to={urlNavigation}>
+            <img
+              src={navigation}
+              alt="avatar"
+              width="50px"
+              height="50px"
+              style={{ borderRadius: '50%' }}
+            />
```

Comment:
Temporary solution, until we merge #92

Reply:
I think you can use the real Avatar component, it has been integrated!